src/briefcase/commands/create.py

```diff
@@ -544,7 +555,17 @@ def _extra_pip_args(self, app: AppConfig):
         :param app: The app configuration
         :returns: A list of additional arguments
         """
-        return []
+        args: list[str] = []
+        for argument in app.requirement_installer_args:
+            if relative_path_matcher.match(argument) and _is_local_path(argument):
+                abs_path = os.path.abspath(self.base_path / argument)
+                if Path(abs_path).exists():
+                    args.append(abs_path)
+                    continue
+
+            args.append(argument)
```

Comment:
A minor code style thing - I try to avoid `continue` and `break` as much as possible, as it's easy to misread the intention of the statement (for a long read example of this happening, [in 1990 almost the entire US phone system was taken down by a stray `break` statement](https://www.mit.edu/hacker/part1.html)).

In a case such as this, I'd lean towards defining a `final_arg` value that is `abs_path` in the continued case here, but set to `argument` otherwise, and then unconditionally appending `final_arg`.

Reply:
Fair enough, thanks for the link to the story!

Done in https://github.com/beeware/briefcase/pull/2059/commits/ca791f43fd7caf445c6387c4f3b1b842204a0809